tests/providers/amazon/aws/operators/test_datasync.py

```diff
@@ -734,6 +818,7 @@ def test_xcom_push(self, mock_get_conn):
         self.set_up_operator()
         ti = TaskInstance(task=self.datasync, execution_date=timezone.utcnow())
         ti.run()
-        self.assertEqual(
-            ti.xcom_pull(task_ids=self.datasync.task_id, key='return_value'),
-            self.task_arn)
+        pushed_task_arn = ti.xcom_pull(
+            task_ids=self.datasync.task_id, key="return_value"
+        )["TaskArn"]
+        self.assertEqual(pushed_task_arn, self.task_arn)
```

Comment:
I think we miss tests for callables.

Reply:
Good catch. I've changed the functionality but still need to add tests regardless, so I'll do that :)